### DISCUSSION TITLE
No option to add google calendar

### DISCUSSION THREAD

**Discussion by IanWish** [kewisch/gdata-provider#402] [Support]
Just downloaded Thunderbird 91.6.0, my google calendars didn't sync or show up.  So I uninstalled provider and re-downloaded and installed the newest version of provider.  When I try and add a calendar, the option to add a Google Calendar is completely missing.

Is this happening to anyone else?  Any fix for this?
Thanks!

EDIT - Just moved to a totally different platform - Mailbird.  The Google calendar integration is baked in, intutive and works flawlessly.  Goodbye Thunderbird/Lightning/Provider!

**Comment by staldershoot** [comment]
Having same issue with 91.6.1...just installed the provider (new install) and I don't see anything about adding a google calendar.

**Reply by iamtheonlydave** [reply]
I can add google calendars but not task lists. BTW Tb is free and Mailbird is expensive. Just saying

**Comment by kewisch** [comment]
Anyone having this issue, it seems like the add-on is not properly loaded. The option should prominently appear. I'd appreciate checking the error console to see what might have failed.
